code/modules/crew_objectives/civilian_objectives.dm

```diff
@@ -200,121 +209,131 @@
-			for(var/datum/feed_message/message in channel.messages)
-				if(ckey(message.author) == ckey(ownername))
-					if(length(message.return_body()) >= charcount)
-						target_amount--
-	if(target_amount <= 0)
+	if(..())
 		return TRUE
-	else
-		return ..()
+	for(var/datum/feed_channel/channel in GLOB.news_network.network_channels)
+		for(var/datum/feed_message/message in channel.messages)
+			if(!istype(message.author_account, /datum/bank_account))
+				continue
+			if(message.author_account.account_id == owner.account_id)
+				if(length(splittext(message.return_body(), " ")) >= word_count)
+					target_amount--
+	return target_amount <= 0
 
 /datum/objective/crew/pwrgame //ported from Goon with adjustments
-	var/obj/item/clothing/targettidegarb
+	var/obj/item/clothing/clothing_target
 	explanation_text = "Get your grubby hands on a (Dear god something broke. Report this on GitHub)."
-	jobs = "assistant"
+	jobs = JOB_NAME_ASSISTANT
 
 /datum/objective/crew/pwrgame/New()
 	. = ..()
-	var/list/muhvalids = list(/obj/item/clothing/mask/gas, /obj/item/clothing/head/welding, /obj/item/clothing/head/ushanka, /obj/item/clothing/gloves/color/yellow, /obj/item/clothing/mask/gas/owl_mask)
+	var/list/possible_targets = list(/obj/item/clothing/mask/gas, /obj/item/clothing/head/welding, /obj/item/clothing/head/ushanka, /obj/item/clothing/gloves/color/yellow, /obj/item/clothing/mask/gas/owl_mask)
 	if(prob(10))
-		muhvalids += list(/obj/item/clothing/suit/space)
-	targettidegarb = pick(muhvalids)
+		possible_targets += list(/obj/item/clothing/suit/space)
+	clothing_target = pick(possible_targets)
 	update_explanation_text()
 
 /datum/objective/crew/pwrgame/update_explanation_text()
 	. = ..()
-	explanation_text = "Get your grubby hands on a [initial(targettidegarb.name)]."
+	explanation_text = "Get your grubby hands on a [initial(clothing_target.name)]."
 
 /datum/objective/crew/pwrgame/check_completion()
-	if(owner?.current)
-		for(var/tidegarbtypes in typesof(targettidegarb))
-			if(owner.current.check_contents_for(tidegarbtypes))
-				return TRUE
-	return ..()
+	if(..())
+		return TRUE
+	if(!owner?.current)
+		return FALSE
+	for(var/search_subtype in typesof(clothing_target))
+		if(owner.current.check_contents_for(search_subtype))
+			return TRUE
+	return FALSE
 
 /datum/objective/crew/promotion //ported from Goon
 	explanation_text = "Have a non-assistant ID registered to you at the end of the shift."
-	jobs = "assistant"
+	jobs = JOB_NAME_ASSISTANT
 
 /datum/objective/crew/promotion/check_completion()
-	if(owner?.current)
-		var/mob/living/carbon/human/H = owner.current
-		var/obj/item/card/id/theID = H.get_idcard()
-		if(istype(theID))
-			if(!(H.get_assignment() == JOB_NAME_ASSISTANT) && !(H.get_assignment() == "No id") && !(H.get_assignment() == "No job"))
-				return TRUE
-			if(theID.hud_state != JOB_HUD_ASSISTANT) // non-assistant HUD counts too
-				return TRUE
-	return ..()
+	if(..())
+		return TRUE
```

Comment:
Why do all of these return true if ..()? What does the parent proc do?
I assume it checks for admin completion, but does it do anything else?

Reply:
Just admin completion